README.md

```diff
@@ -134,6 +134,14 @@ The code above uses a test web sockets server from "PieSocket", you can run the
 and go to [their test page](https://www.piesocket.com/websocket-tester) to send messages.
 The widget will automatically update to the latest data sent through the socket.
 
+### MqttString
+
+A `MqttString` binding creates a `String` data binding to the specified _topic_ associated with
+the specified **MQTT** client connection. Each time a message is received the value will be converted
+to a `string` and set on the binding. Each time the value is edited, it will be sent back over
+**MQTT** on the specified _topic_. It is also a `Closable` so you should be sure to call `Close`
+once you are completed using it to disconnect the _topic_ handler from the **MQTT** client connection.
+
```

Comment:
Would it be possible to add a little usage info or demo in here, just make it a bit clearer how to use?